error：
 ! [rejected]        main -> main (fetch first)
error: failed to push some refs to 'https://github.com/hakuya0721/learning-for-python.git'
hint: Updates were rejected because the remote contains work that you do not
hint: have locally. This is usually caused by another repository pushing to
hint: the same ref. If you want to integrate the remote changes, use
hint: 'git pull' before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.

git push -u origin main --force 
强制将本地的覆盖库中

error：
按钮不可用，一直卡着

git add .
git commit -m "修复登录页面按钮样式问题"